packages/web/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@vercel/speed-insights",
-  "version": "1.0.4",
+  "version": "1.0.5-beta.1",
```

```suggestion
  "version": "1.0.5",
```
Lets not forget to update the version before merging